expressjs/test/safepaths.js

```diff
@@ -91,33 +92,30 @@ describe('Safe Path ', function() {
   describe('GET /safe_path with redacted_trails and without publication', function() {
 
```

Comment:
We can put the identifier in this scope, so it can be used for both saving and retrieval. Then, we can substitute `identifier` into where it is used. Currently, it is hard-coded in the tests. See below:

![image](https://user-images.githubusercontent.com/48393820/82610333-2d8a8180-9b8c-11ea-855c-0b1a5f79a0ab.png)


```suggestion
const trailIdentifier = 'a88309c1-26cd-4d2b-8923-af0779e423a3';
```